roles/openshift_buildoverrides/vars/main.yml

```diff
@@ -4,7 +4,7 @@ buildoverrides_yaml:
     configuration:
       apiVersion: v1
       kind: BuildOverridesConfig
-      forcePull: "{{ openshift_buildoverrides_force_pull | default('', true) }}"
+      forcePull: "{{ openshift_buildoverrides_force_pull | default('',true) | bool }}"
```

Comment:
previous example was flawed because of mangling the syntax...  "{{ true if (openshift_build_overrides_force_pull | default(false) | bool else '' }}" should have worked, but wouldn't differentiate unset from false.